webnode/src/redux/index.js

```diff
@@ -1,7 +1,6 @@
 import { createStore, compose, applyMiddleware } from "redux";
 import { createLogger } from "redux-logger";
 import { createEpicMiddleware } from "redux-observable";
-import thunk from "redux-thunk";
```

Comment:
lets remove thunk from the dependencies too

Reply:
good call